src/commands/convert/index.ts

```diff
@@ -94,6 +102,22 @@ export default class Convert extends FingerprintingConvertCommand {
       break
     }
 
+    case 'prisma': {
```

Comment:
Is this in the SAF readme?

Reply:
Yup! It's in a previous PR that added the prisma2hdf converter as a supported format - this PR is just to add it to the generic convert command, which previously didn't have a case to handle it. prisma2hdf is also a separate command. The ReadME isn't updated yet in the main branch, but will be updated as part of a separate PR

https://github.com/mitre/saf/pull/461/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
Perfect. Thanks!